tests/test_validate_metadata.py

```diff
@@ -8,50 +9,140 @@
 
 
 class TestValidateMetadata(unittest.TestCase):
-    def setup_validation(self, args):
+    def setup_metadata(self, args):
         args_list = args.split(' ')
         args = create_parser().parse_args(args_list)
+        with open(args.convention, 'r') as f:
+            convention = json.load(f)
         filetsv = args.input_metadata
         metadata = CellMetadata(filetsv)
         metadata.validate_format()
-        return metadata
+        return (metadata, convention)
 
-    def test_format_name(self):
-        """Header row of metadata file should start with NAME keyword
+    def teardown_metadata(self, metadata):
+        metadata.filehandle.close()
+
+    def test_header_format(self):
+        """Header rows of metadata file should conform to standard
         """
 
-        args = ('../tests/data/AMC_v0.8.json ' '../tests/data/error_test10.tsv')
-        metadata = self.setup_validation(args)
+        args = (
+            '../tests/data/AMC_v0.8.json '
+            '../tests/data/error_headers.tsv'
+        )
+        metadata = self.setup_metadata(args)[0]
         self.assertFalse(metadata.validate_header_keyword())
-
         self.assertIn(
             'Error: Metadata file header row malformed, missing NAME',
             metadata.errors['format'],
-            "Validation should detect malformed NAME keyword"
+            'Missing NAME keyword should fail format validation'
```

Comment:
This phrasing -- reporting how the code under test doesn't meet expectations -- is a helpful refinement.